src/tests/mldsa.rs

```diff
@@ -0,0 +1,449 @@
+         21AAEE94F5252FAD9E01652025F1678BE6044FAEBBA8E94DC43AF8902A5058CE\
+         29D6E17325B1F13A0FB3EEFE46067EEE5BF8E462CB9B6B466B14C541BD211E62\
+         60A31041C748EFDF1B35E75B0133E99283206344598ACEE2AD92519A90E99FD9\
+         6DFEECB58BB40B2ADCA1B462E910D394181F9DF9B1D92686BA21E243DB7D8FE2\
+         A0713D6CCCBC0328F3F9DBCB3037DCF9E811F8C25B0AB9E73D4BCF1335187A86\
+         EE8911BB494B8F1B0228803EA37C8EFD3D947A08226F4A262984C20A9E58558E\
+         6AE03FA694622A634594DB35CC41974DF43C12D042DD752EAE99F31D34BB52AE\
+         D1DA0069568E1C7A1EE725A7AED750DE79D94D7AB75499019D6C2DDA120A1C6F\
+         813AC008B16CAED0015AE609818FE7D8AD1EDD247DA3153EE3D9C8836C77EEC8\
+         4B885F3856DFC304B57C67D930BCD6C77A29271F9035EA6212A9DCF5526DAF19\
+         C47533BF80BB6D15ECB582A78A67BA92F5B413E17AE69C2EFF3BC2EBF5256E92\
+         832130D3EC7F7AD76CC8C5DD59FC2EE9E0A7873806C152791741E6C6533F6368\
+         506FC4A655A92CF77FC44BD4CE3A4480D588DCAB0C8B398AE8CA6A7DA65792C3\
+         946102C45DF4C72EC9FAFF22430F131EDE726DC2987B4DABB72AFE5376A9798C\
+         FF3E8758B3F7BB56692BCC4822CA337395B6C2540FE966A70AC469FF6DDF0909\
+         AB83434C34A854381F5BADF46E4DB78A317E0BCA59A3A8C5972F8D059D7B3A51\
+         470D560D3B47EBC7FA13A253662305B78F7A146FE86B4381B4EE360ADB027C96\
+         46BBCEDEB03C959E4F3111B2F8434949CBF04CD487BBD100708F47C12715E4BC\
+         9E9D87DA5717246E6F3E4354636CE312560673F4037E4651A2098553C64C11BC\
+         FC9B881C7858583066D80353ABA92869BB20B75106C233619B95D7EC58630CC2\
+         D4158650582C345D3C3110361DDE04B69F38175D136F1E28020D36D2D90F2853\
+         748A232BAEEDEC38A739781BFB7F52154BA20DF52BEBA4CD65ABC7DC57EC4845\
+         65186DAA795B42C8988293B2D4EF0524CBDE85143ADA3D1B28374D2B1A335C6A\
+         914705CBC2E2DBA7D363ED78590369FF256B3BE230412E821407865A07D1DA11\
+         7BAE75B0226D173C14B15B3E85AE50A569ABD465D4E4500FDF535BBC2C1C5AE9\
+         C1CA73753CF488A9A4BB406549BD3BE25AFC62F7B70DFC2E940B8C37CF33D2F8\
+         0AB3D1070B03617043BABF7D687CB04B33AAEE8005F58D06C4464440875A79AF\
+         4F5079FFA1CCA1B109419223FF9E56674D834FBFDE69F406873C3E54218EFD8F\
+         70D3DB7084AE9734E31906AD94491CE12B828CC785080DCC911ACF80C522817C\
+         CAEF838BC50ECBB499F12ABF32647C425C7A432C0321F13EF4E18B628DC68D7B\
+         D5D049592A73A96C2CC8F2D4924A1D14ABB2B15DB7BD5BD4A46B09FAFB0F8057\
+         8C2B8D0F4767EE26AB05D22A3BF2F02FCDA404646495F27B2A4E6BFD33E73201\
+         3DE703CAE16B45F9BC39BA8FA2EE8FF427DA4FAC6C822B905465014647873E1E\
+         E8832A87870107B9856E950EEDC6FB097F5383DDF6E4AE01C98D5AC6054AB356\
+         55318517E669E379140246AE63D52066037FF37B6814A911A15C0982140DC132\
+         143F8707C04C6F81742872FA83E6CFA8112C644BC1A86162527F924D9DF22341\
+         232E3540484956677D99D6DFFB055A7A7F8C9EAFB9BBC4C8D2D7DADEDFFA1B2B\
+         476578798091B8B9E2F0F1FBFE090B2E47818CAECED6E0E9FC00000000000000\
+         000000000000000000000000000000000D1E2D39",
```

Comment:
the zeroes at the end are fishy, but it looks like this is how signatures look like in ML-DSA ...

Reply:
The last 64 bytes are a nonce IIRC.